packages/main/src/themes/StepInput.css

```diff
@@ -122,11 +121,11 @@
 }
 
 :host .ui5-step-icon.ui5-step-dec {
-	left: 0;
+	left: var(--_ui5_step-input_button_left);
 }
 
 :host .ui5-step-icon.ui5-step-inc {
-	right: 0;
+	right: var(--_ui5_step-input_button_right);
 }
 
 :host .ui5-step-icon *:not([clickable]),
```

Comment:
In openUI5 when the increase/decrease button is active, there is a box-shadow and I think that it is the expected behaviour according to the requirement for the other inputs. Check the visualisation in openUI5 and make it the same here, when the button is active.

Reply:
If you hover over the increase/decrease button, the component box-shadow in not visible. When you hover over the buttons, it should be the same as you hover over the input field.

Reply:
done